baselines/audioworklet.generated.d.ts

```diff
@@ -216,6 +216,15 @@ declare var AudioWorkletProcessor: {
     new(): AudioWorkletProcessor;
 };
 
+interface AudioWorkletProcessorImpl extends AudioWorkletProcessor {
+    process(inputs: Float32Array[][], outputs: Float32Array[][], parameters: Record<string, Float32Array>): boolean;
+}
+
+declare var AudioWorkletProcessorImpl: {
```

Comment:
It doesn't really exist right?

Reply:
yes how do I prevent this?

Reply:
Not sure. Try removing `"exposed": "AudioWorklet"` in override?

Reply:
noInterfaceObject will do the work.